indy-repository-manager/src/main/java/org/jboss/pnc/indyrepositorymanager/IndyRepositorySession.java

```diff
@@ -666,7 +669,7 @@ private boolean ignoreContent(String packageType, String path) {
                 suffixes = ignoredPathSuffixes.getNpmWithShared();
                 break;
             case GENERIC_PKG_KEY:
-                suffixes = ignoredPathSuffixes.getShared();
+                suffixes = ignoredPathSuffixes.getGenericWithShared();
```

Comment:
I would stick to Shared only, because if we have a missing package type with some specific metadata path patterns we want to ignore, we shouldn't put them here, because it could cause issue with other unsupported package types.